source/renderer/dialog/preferences.js

```diff
@@ -19,19 +19,24 @@ const validate = require('../../common/validate.js')
 const { ipcRenderer } = require('electron')
 const { trans } = require('../../common/lang/i18n')
 const generateId = require('../../common/util/generate-id')
+const { renderTemplate } = require('../util/template.js')
 
 class PreferencesDialog extends ZettlrDialog {
   constructor () {
     super()
     this._dialog = 'preferences'
     this._boundCallback = this.afterDownload.bind(this)
     this._textTimeout = null
+  }
 
-    // Build the loading spinner that we need for the downloading indication
-    this._spinner = $('<div>').addClass('sk-three-bounce')
-    for (let i = 1; i < 4; i++) {
-      this._spinner.append($('<div>').addClass('sk-bounce' + i).addClass('sk-child'))
-    }
+  get spinner () {
```

Comment:
Is this function in the constructor …?

Reply:
No, it is outside of the constructor and it is a property getter on the class.